src/rpc/register.h

```diff
@@ -10,23 +10,23 @@
 class CRPCTable;
 
 /** Register block chain RPC commands */
-void RegisterBlockchainRPCCommands(CRPCTable &tableRPC);
+void RegisterBlockchainRPCCommands(CRPCTable &t);
```

Comment:
In prototypes, you do not need argument names at all, so I expect the change is not needed strictly speaking. I did it just for consistency.